From 90fbb4e5c58b0b2c484bf109d0d6e585e5947f94 Mon Sep 17 00:00:00 2001
From: Saleem Abdulrasool <compnerd@compnerd.org>
Date: Sun, 5 Oct 2008 15:01:17 -0700
Subject: [PATCH] partial fix for bug #21368

Links should not be mouse focusable.  This should fix the 2px border that
appears when links are clicked.
---
 WebCore/html/HTMLAnchorElement.cpp |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/WebCore/html/HTMLAnchorElement.cpp b/WebCore/html/HTMLAnchorElement.cpp
index e4d10d1..baae605 100644
--- a/WebCore/html/HTMLAnchorElement.cpp
+++ b/WebCore/html/HTMLAnchorElement.cpp
@@ -91,6 +91,9 @@ bool HTMLAnchorElement::isFocusable() const
 bool HTMLAnchorElement::isMouseFocusable() const
 {
 #if PLATFORM(GTK)
+    if (isLink())
+       return false;
+
     return HTMLElement::isMouseFocusable();
 #else
     return false;
-- 
1.6.0.2

